### ISSUE TITLE
[iOS] how to pan gesture outside of the globe 

### ISSUE THREAD

**Issue by codaman** [mousebird/WhirlyGlobe#1222]
In 3D Map, I want to rotate globe outside of globe. Pan is just working in globe area. I tried many things to pan outside of the globe.

**Comment by mousebird** [comment]
Do you mean you want the pan gesture to work when the user has moved their finger outside of the globe area?
If so, I don't have a switch for that.

**Comment by codaman** [comment]
Yes i want to add this function. I guess this function more useful for user experience. I saw many users try to turn globe not only inside of the globe but also outside

**Comment by mousebird** [comment]
The current pan gesture moves a certain point on the globe by keeping it underneath the active finger.  You'd need to do something else outside the globe if you chose to.

**Comment by andrewhoyer** [comment]
Adding my thoughts here. It's an interesting idea, but it should be an activated feature, not default, to maintain expected functionality. As it happens, I've been thinking about using areas like those outside the globe as a way to hide/show the HUD and/or other page elements. Example: Tap inside the globe to move to that location, tap outside the globe to toggle HUD. Right now I believe I could capture those taps, but adding other things like swipes might affect the way that works.
